.changeset/seven-nails-relate.md

```diff
@@ -0,0 +1,5 @@
+---
+'@liteflow/core': minor
```

Comment:
Keeping it minor change, as updating import path's are needed for the `@liteflow/core` package where we use `BatchPurchaseState`